src/test/java/org/apache/sysds/test/functions/io/csv/ReadCSVTest1.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.sysds.test.functions.io.csv;
+
+public class ReadCSVTest1 extends ReadCSVTest {
+
+	private final static String TEST_NAME = "ReadCSVTest";
+	private final static String TEST_CLASS_DIR = TEST_DIR + ReadCSVTest1.class.getSimpleName() + "/";
+
+	protected String getTestName() {
+		return TEST_NAME;
+	}
+
+	protected String getTestClassDir() {
+		return TEST_CLASS_DIR;
+	}
+
+	protected int getId() {
+		return 1;
+	}
+}
```

Comment:
I preferred the old version, although it could have used arrays for the IDs/test_numbers and parameterized tests. It comes down to personal preference tough. Same with the other tests.

Reply:
I preferred the old as well for syntax reasons, but not for performance reasons (not really a good argument i guess). When i extend like this the test can run in parallel, before they did not, and since i intended at the time to extend the CSV Testing in the next commits i made the changes here.

Then again this is a roundabout way of fixing it in general, because it is still using the awkward AutomatedTestBase and setup, that to make this really cool needs to be rethought.

for reference when executing:
`mvn surefire:test -DskipTests=false -Dtest=org.apache.sysds.test.functions.io.csv.**`
it went from total time of `22.836 s` to `16.548 s`,

Reply:
Performance actually seems like a good argument, since we have sooo many tests.

Okay, so they will get another rework? It is not really bad, although changes now affect quite a few classes, but tests anyway are not overworked as often so it should be fine, right?

Yes, the `AutomatedTestBase` really suffers from legacy code, maybe we could select some standard and mark some methods deprecated.

Reply:
yes deprecation markings could be a way forward.